torch_xla/csrc/ops/einsum_backward.cpp

```diff
@@ -0,0 +1,145 @@
+#include "torch_xla/csrc/ops/einsum_backward.h"
+
+#include "tensorflow/compiler/xla/client/lib/matrix.h"
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/einsum.h"
+#include "torch_xla/csrc/ops/infer_output_shape.h"
+#include "torch_xla/csrc/ops/xla_ops.h"
+
+namespace torch_xla {
+namespace {
+
+absl::Span<const xla::XlaOp> BuildEinsumBackward(
```

Comment:
this seems weird, Span is a reference of vector but the vector you are referring to is a local stack variable  in this function. By the time caller access span the variable is already gone and will cause invalid access. I think you can just return a vecotr, pair or tuple of `xla::XlaOp`

Reply:
Done